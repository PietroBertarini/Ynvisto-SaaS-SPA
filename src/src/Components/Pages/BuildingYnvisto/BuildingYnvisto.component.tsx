import React from 'react';
import Referral from '../../Components/Referral/Referral.component';
import { ReactComponent as Social } from '../../../Assets/Images/social-growth.svg';
import { Main } from './BuildingYnvisto.styles';

const BuildingYnvisto: React.FC = () => {
  const title = 'O que você gostaria de ver na Ynvisto ?';
  const text = ['Nós da Ynvisto gostaríamos de facilitar a vida do investidor brasileiro '
   + 'e faremos isso com base na opinião de cada cliente nosso, construindo um produto que seja '
   + 'referência no mercado.', 'Com isso, gostaríamos de escutar cada pedido de nossos clientes, '
   + 'para que seja algo feito para a comunidade da Ynvisto.'];
  const buttonTitle = 'Faça seu pedido aqui';

  return (
    <Main>
      <Referral
        title={title}
        texts={text}
        svg={(
          <Social
            style={{ height: '50vh' }}
          />
        )}
        buttonTitle={buttonTitle}
        onClick={() => {
          window.open('https://form.jotform.com/ynvisto/construa-ynvisto');
        }}
      />
    </Main>
  );
};

export default BuildingYnvisto;
