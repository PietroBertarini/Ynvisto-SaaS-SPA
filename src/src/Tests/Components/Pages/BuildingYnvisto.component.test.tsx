import React from 'react';
import { shallow } from 'enzyme';
import BuildingYnvisto from '../../../Components/Pages/BuildingYnvisto/BuildingYnvisto.component';

test('Renders BuildingYnvisto component and match to previous snapshot', () => {
  const component = shallow(<BuildingYnvisto />);
  expect(component).toMatchSnapshot();
});
