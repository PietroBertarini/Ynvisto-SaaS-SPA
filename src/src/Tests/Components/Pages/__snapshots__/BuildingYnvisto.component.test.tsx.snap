// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders BuildingYnvisto component and match to previous snapshot 1`] = `
<styled.div>
  <Referral
    buttonTitle="Faça seu pedido aqui"
    onClick={[Function]}
    svg={
      <ForwardRef(SvgSocialGrowth)
        style={
          Object {
            "height": "50vh",
          }
        }
      />
    }
    texts={
      Array [
        "Nós da Ynvisto gostaríamos de facilitar a vida do investidor brasileiro e faremos isso com base na opinião de cada cliente nosso, construindo um produto que seja referência no mercado.",
        "Com isso, gostaríamos de escutar cada pedido de nossos clientes, para que seja algo feito para a comunidade da Ynvisto.",
      ]
    }
    title="O que você gostaria de ver na Ynvisto ?"
  />
</styled.div>
`;
